app/models/label/LabelTable.scala

```diff
@@ -691,9 +985,36 @@ object LabelTable {
       case e: SocketException => false
       case e: Exception => false
     }
-
   }
 
+  // /**
```

Comment:
Was this just replaced by the `panoExists()` function?

Reply:
Yeah, I think so? I don't recall writing any of this though. I'll get rid of it anyways since it's literally the same thing.